index.js

```diff
@@ -18,6 +19,11 @@ module.exports = function markdownItPandoc(md, markdownItPandocOpts) {
                }
              , markdownItPandocOpts)
 
+  // highlightjs must be enabled before attributes
+  if (opts.highlightjs) {
+    md = md.use( require('markdown-it-highlightjs'), {code: false,inline: true} );
```

Comment:
you think the `inline: true` is a better default? I think pandoc also doesn't highlight inline code blocks by default, or does it?

Reply:
Honestly Mauro, everything I know about pandoc I learned from working on panwriter over the past few days. :)

I don't feel strongly about inline. Actually, I realized after I used it for a while that it seemed unusual for the inline code to be highlighted.

I'll remove it. (Although, I _am_ interested in how you'd like settings like this to be overridden/configured because I hope to move the latex delimiters in here next.)

Reply:
Cool!

> I am interested in how you'd like settings like this to be overridden/configured because I hope to move the latex delimiters in here next.

We'd need to pass some part of `opts` along, like:

```js
  if (opts.katex && !opts.mathjax) {
    md = md.use(
      require('markdown-it-texmath').use({
        engine: require('katex'),
        delimiters: opts.tex_math_single_backslash ? 'brackets' : 'dollars',
      })
    );
  }
```

Reply:
Ah! I missed that the opts passed into markdown-it-pandoc are _merged_ into the peer dep opts!

OK, got it! Thanks!

Reply:
(so we'd just read all opts from the settings.yaml and always pass them through to markdown-it-pandoc...)

Reply:
yes, probably it makes sense to pass `settings.extensions` into `markdown-it-pandoc` as is.